### ISSUE TITLE
MG + MIS - Data Searching Module Has Multiple Secrets

### ISSUE THREAD

**Issue by rnorouziani** [CynthiaLiu0805/BridgeCorrosion#60]
The Data Searching Module, due to its functionality of reading from a database, should have the database structure as one of its secrets. This indicates that the module has multiple secrets: the database structure and the process of the data.

**Comment by CynthiaLiu0805** [comment]
You are right, the data searching module do need to know about the database structure, it is fixed in [1cbcc74](https://github.com/CynthiaLiu0805/BridgeCorrosion/commit/1cbcc74e04a5cad997aafccaa6df3b5fe5ea7d83)
